README.md

```diff
@@ -19,6 +19,17 @@ Getting started
 1. Run `make`
 2. Run `./start.sh`
 
+### Usage
+In order to use the generator, there are 2 possible endpoints that can be used. 
+
+- POST: Sending a POST request on the following endpoint `{Base URL}/test-data/company)` will generate a new test company and accompanying Authcode.
+	- Optional: CompanySpec when added to the request body will alter generated company to be based in Scotland or Northern Ireland but will default to England/Wales.
+		- Usage example: `{"jurisdiction":"scotland"}`
```

Comment:
I personally don't like and find confusing the fact that we use 3 levels of listing when there is nothing to list but up to you!